enterprise/server/remote_execution/containers/sandbox/sandbox_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build darwin
```

Comment:
```suggestion
//go:build darwin && !ios
```

This should help you simplify the generated BUILD file for this package.
We are never building our binary targeting iOS anyway 🤔

Reply:
you might need to re-run Gazelle locally after applying the suggestion.